actionpack/lib/action_controller/renderer.rb

```diff
@@ -99,6 +99,7 @@ def render(*args)
       instance.set_response! controller.make_response!(request)
       instance.render_to_string(*args)
     end
+    alias_method :render_to_string, :render
```

Comment:
Why this alias is being added?

Reply:
To allow either `ActionController::Renderer` or [`AbstractController::Base`](https://github.com/rails/rails/blob/357013d36698f3e10ea935cf31f7cad426b785ce/actionpack/lib/abstract_controller/rendering.rb#L45) to be used for the [`renderer`](https://github.com/rails/rails/pull/40222/files#diff-2925f29eb936dbc8864edeee665ef2fbR25).

Reply:
But do we want that to be part of the public API or not? I don't need any reason for that, so maybe we should `nodoc` the alias?